main.rb

```diff
@@ -26,6 +31,8 @@ def main_menu
       CreateFile.new.call
     when 3
       UpdateFile.new.call
+    when 4
+      FilterFile.new.call
```

Comment:
Try to use [Factory](https://medium.com/sudo-by-icalia-labs/i-discovered-the-factory-pattern-earlier-this-year-e68a44572ca7) pattern here. 
```ruby
INPUT_TO_ACTION_MAPPER = {
  1 => :read,
  2 => :create,
  3 => :update,
}

input = gets.chomp.to_i
YourFactory.call(INPUT_TO_ACTION_MAPPER[input]).perform

class YourFactory
  ACTIONS = {
    :read => ReadClass,
   :create => CreateClass
  }
  
  def self.call(action_name, params)
    action = ACTIONS.fetch(action_name)
    # here you can implement fetch(action_name, FallbackClass/NullAction)
    # https://medium.com/@kelseydh/using-the-null-object-pattern-with-ruby-on-rails-b645ebf79785
    # that will print something with delay and exit
    action.new(params)
  end
end
```

Reply:
Created FileFactory